android/src/main/java/io/mosip/tuvali/common/safeExecute/TryExecuteSync.kt

```
@@ -12,7 +13,7 @@ class TryExecuteSync(private val bleExceptionHandler: OpenIdBLEExceptionHandler)
       try {
         returnValue = fn()
       } catch (e: Exception) {
-        bleExceptionHandler.handleException(Exception("Unknown Exception", e))
+        bleExceptionHandler.handleException(UnknownException("Caught unknown exception in Try Execute", e))
```

We should throw exception as-is, Converting to UnknownException can be avoided